src/Surfnet/Stepup/Identity/Entity/VettedSecondFactor.php

```diff
@@ -89,6 +93,24 @@ public function getId()
         return $this->id;
     }
 
+    public function move(Identity $identity, MoveTokenVettingType $type)
+    {
+        $this->apply(
+            new SecondFactorVettedEvent(
```

Comment:
I would strongly suggest to use a dedicated `MoveSecondFactorEvent` or `CopySecondFactorEvent`, because the token seems to get copied and not vetted after all. It would then be more easily to apply only logic for copying a token. Sending mails to inform the end user the token was successfully vetted for example doesn't seem in place here. So in the end additional logic could also be added without hassle and without a lot of exceptions in the code.